flask/app/app/views.py

```diff
@@ -7,7 +7,13 @@
 @app.route('/')
 @app.route('/index')
 def index():
-    if 'REDIS_PORT' in os.environ:
+    if 'REDIS_PORT' in os.environ and 'REDIS_HOST' in os.environ:
+        redis_port = os.environ['REDIS_PORT']
+        redis_host = os.environ['REDIS_HOST']
+        r = redis.StrictRedis(host=redis_host,
+                              port=redis_port,
+                              db=0)
+    elif 'REDIS_PORT' in os.environ:
```

Comment:
as per our conversation off github, instead of adding new code to `elif` I added it to first `if` because there is overlap of environment `REDIS_PORT`.

When using docker(nulecule), k8s(nulecule); `REDIS_PORT` is available while using docker-compose is `REDIS_PORT` and `REDIS_HOST` both are available, which are injected by the compose file as shown in https://github.com/dharmit/flask_redis_nulecule_app/pull/1#issuecomment-217861525